src/renderer/features/fellowship-profile/model/profile.ts

```diff
@@ -28,6 +30,14 @@ const $identity = combine($currentMember, $identities, (member, identities) => {
   return identities[member.accountId] ?? null;
 });
 
+const $isAccountExist = profileFeatureStatus.input.map(store => {
+  if (!store) return false;
+
+  return store.accounts.some(account => {
+    return !accountUtils.isBaseAccount(account) && accountUtils.isChainAndCryptoMatch(account, store.chain);
```

Comment:
Looks like check on PV wallet should be added with `!accountUtils.isBaseAccount(account)`